test/sagemaker_tests/pytorch/training/integration/sagemaker/test_health_check.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
These tests need to be shifted to the sanity test suite. Please place them within [this](https://github.com/aws/deep-learning-containers/tree/master/test/dlc_tests/sanity) folder.

Also, as of now, these test have not run on the CICD since these are not in the sanity test suite. Once the tests are shifted to sanity test folder, these would start to run.

Do we want to run these tests on a specific instance type? We would need to add an `ec2_instance` parameter in that case and follow the approach similar to this: https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/sanity/test_telemetry.py#L27-L31.